platform/error/platform-error-impl/src/main/resources/error.html

```diff
@@ -0,0 +1,128 @@
+<!--
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+-->
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
+<head>
+  <title>Error</title>
+  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+  <style>
```

Comment:
📓 Per the advice of @djblue, I inlined the styles here. I believe I've reduced them sufficiently, but if there are any others that should be removed, I would be happy to make this smaller still.